runtime.md

```diff
@@ -33,24 +33,79 @@ This is provided so that consumers can find the container's configuration and ro
 }
 ```
 
-## Lifecycle
+## Operations
+
+OCI compliant runtimes MUST support the following operations, unless support for the operation can not be supported by the base operating system (kernel) itself.
```

Comment:
On Wed, Oct 14, 2015 at 10:55:52AM -0700, Mrunal Patel wrote:
> > +OCI compliant runtimes MUST support the following operations,
> > unless support for the operation can not be supported by the base
> > operating system (kernel) itself.
> 
> Maybe simplify a bit to "unless the operation is not supported by
> the base operating system" ?

+1 to Mrunal's wording, but I'd switch “base” → “host” as well.